src/lib/openapi/spec/export-result-schema.ts

```diff
@@ -166,6 +168,14 @@ export const exportResultSchema = {
                 $ref: '#/components/schemas/tagTypeSchema',
             },
         },
+        featureDependencies: {
```

Comment:
I'd call this dependencies since it opens a way for other types of dependencies (e.g. strategy dependencies)